assets/src/edit-story/theme.js

```diff
@@ -116,6 +116,7 @@ const theme = {
       black: '#000000',
       white: '#FFFFFF',
       primary: '#EDEFEC',
+      secondary: '#A1A09B',
```

Comment:
I got this name and color from the [figma doc](https://github.com/google/web-stories-wp/pull/4355).